Source/Core/VideoCommon/CommandProcessor.cpp

```diff
@@ -254,20 +254,20 @@ void CommandProcessorManager::RegisterMMIO(Core::System& system, MMIO::Mapping*
   MMIO::ReadHandlingMethod<u16>* fifo_rw_distance_lo_r;
   if (is_on_thread)
   {
-    fifo_rw_distance_lo_r = MMIO::ComplexRead<u16>([](Core::System& system, u32) {
-      const auto& fifo = system.GetCommandProcessor().GetFifo();
-      if (fifo.CPWritePointer.load(std::memory_order_relaxed) >=
-          fifo.SafeCPReadPointer.load(std::memory_order_relaxed))
+    fifo_rw_distance_lo_r = MMIO::ComplexRead<u16>([](Core::System& system_, u32) {
+      const auto& fifo_ = system_.GetCommandProcessor().GetFifo();
```

Comment:
And `fifo`... in the parent scope, it's just an alias to `m_fifo`

Can we change those instead to all use `m_fifo`?